Jenkinsfile

```diff
@@ -53,38 +53,6 @@ pipeline {
 				}
 			}
 		}
-		stage('Build Docs') {
-			steps {
-				withCredentials([string(credentialsId: 'gpg-passphrase', variable: 'KEYRING_PASSPHRASE')]) {
-				wrap([$class: 'Xvnc', useXauthority: true]) {
-					sh 'cd mylyn.docs && mvn clean verify -B -Psign -Dmaven.repo.local=$WORKSPACE/.m2/repository -Dmaven.test.failure.ignore=true -Dmaven.test.error.ignore=true -Ddash.fail=false -Dgpg.passphrase="${KEYRING_PASSPHRASE}"'
```

Comment:
And how do we build Mylyn Docs after this change @BeckerFrank ?

Reply:
Good catch, by mistake I did not include the change in org.eclipse.mylyn/pom.xml.